tools/c7n_tencentcloud/tests/test_tc_tags.py

```diff
@@ -0,0 +1,303 @@
+                ],
+                "actions": [
+                    {
+                        "type": "terminate"
+                    }
+                ]
+            },
+            config=options
+        )
+        policy.run()
+
+        assert len(cvm.resources()) == 0
+
+    @pytest.mark.vcr
+    def test_add_tag(self, options):
+        resources = self.cvm.resources()
+        for tag in resources[0]["Tags"]:
+            assert tag["Key"] != "tag_add_test_key_for_test"
+
+        policy = self.load_policy(
+            {
+                "name": "cvm-test-tag",
+                "resource": "tencentcloud.cvm",
+                "filters": [
+                    {
+                        "InstanceId": "ins-00lycyy6",
+                    }
+                ],
+                "actions": [
+                    {
+                        "type": "tag",
+                        "key": "tag_add_test_key_for_test",
+                        "value": "tag_add_test_value_for_test"
+                    }
+                ]
+            },
+            config=options
+        )
+        policy.run()
+
+        resources = self.cvm.resources()
+        tag_add_success = False
+        for tag in resources[0]["Tags"]:
+            if tag["Key"] == "tag_add_test_key_for_test" and \
+                    tag["Value"] == "tag_add_test_value_for_test":
+                tag_add_success = True
+                break
+        assert tag_add_success
+
+    @pytest.mark.vcr
+    def test_modify_tag(self, options):
+        resources = self.cvm.resources()
+        tag_exist = False
+        for tag in resources[0]["Tags"]:
+            if tag["Key"] == "tag_add_test_key_for_test":
+                tag_exist = True
+                break
+        assert tag_exist
+
+        policy = self.load_policy(
+            {
+                "name": "cvm-test-rename-tag",
+                "resource": "tencentcloud.cvm",
+                "filters": [
+                    {
+                        "InstanceId": "ins-00lycyy6",
+                    }
+                ],
+                "actions": [
+                    {
+                        "type": "rename-tag",
+                        "old_key": "tag_add_test_key_for_test",
+                        "new_key": "tag_add_test_key_for_test_rename"
+                    }
+                ]
+            },
+            config=options
+        )
+        policy.run()
+
+        old_key_not_exist = True
+        new_key_exist = False
+        resources = self.cvm.resources()
+        for tag in resources[0]["Tags"]:
+            if tag["Key"] == "tag_add_test_key_for_test":
+                old_key_not_exist = False
+                break
+            if tag["Key"] == "tag_add_test_key_for_test_rename":
+                new_key_exist = True
+        assert old_key_not_exist and new_key_exist
+
+    @pytest.mark.vcr
+    def test_remove_tag(self, options):
+        resources = self.cvm.resources()
```

Comment:
remove this, and add it as a filter in the policy, that removes a set of recorded resources.